com.mudeprolinux.whakarere.yml

```diff
@@ -29,4 +29,4 @@ modules:
       - type: git
         url: https://github.com/tobagin/whakarere
         tag: 0.0.1
-        commit: 819655cbaa74d6af1eecae1ca66516c269cfa16b
\ No newline at end of file
+        commit: 60205d45948849a2938fc8ffe36e200d903fdd8b
```

Comment:
retagging is bad.

Reply:
thanks, i did not know that. as you can see i'm new to this whole git thing.